Dockerfile

```diff
@@ -15,4 +15,6 @@ ADD scripts/start.sh /
 # Test certifiates will be overwritten in production by configmap
 ADD test/certs /etc/nginx/certs
 
+RUN chown -R nginx.nginx /www
```

Comment:
@glitchcrab Is it really `nginx.nginx`, not `nginx:nginx`?

Reply:
Either way works, but I guess most people are used to the latter so I'll change it. This Dockerfile requires more work to run properly anyway, it's a bit of a WIP.